articles/azure-vmware/create-placement-policy.md

```diff
@@ -10,9 +10,6 @@ ms.date: 8/18/2021
 
 # Create a placement policy in Azure VMware Solution (Preview)
```

Comment:
@ashwinkabadi did you mean to remove (Preview) for the title here?

Reply:
Yes, as I stated in the meeting, this feature will be GA next week. So, we should take out (Preview) from the title